frontend/sw360-portlet/src/main/resources/META-INF/resources/html/utils/includes/editAttachments.jspf

```diff
@@ -180,6 +190,23 @@
             }
         });
 
+        let functionForCheckStatusOfInitialScanType = function (attachmentType, checkStatus) {
+            if(attachmentType == "18") {
```

Comment:
Indentation after if, same for line 203.

Reply:
Fixed